src/Toggle/Toggle.js

```diff
@@ -120,6 +120,10 @@ class Toggle extends Component {
      */
     thumbStyle: PropTypes.object,
     /**
+    * Override the inline styles for thumb when the toggle switch is toggled on.
+    */
+    thumbWhenSwitchedStyle: PropTypes.object,
```

Comment:
@oliviertassinari do you think we should remove the `When` from these props? They are rather verbose